tests/bsim/bluetooth/mesh/CMakeLists.txt

```diff
@@ -29,10 +29,21 @@ if(CONFIG_SETTINGS)
  if(CONFIG_BT_MESH_V1d1)
   target_sources(app PRIVATE
    src/test_dfu.c
-   src/test_blob.c
    src/test_sar.c
    src/test_lcd.c
- )
+   )
+
+  if(CONFIG_BT_MESH_PROXY_SOLICITATION)
```

Comment:
For the proxy sol test I agree. I can remove this added if-statement and place it under GATT_PROXY AND MESH_V1d1. However, I am struggling to place the blob test if I am to remove the added FRIEND-check. If left where it was, I get errors indicating that FRIEND is not enabled. Any suggestions?

Reply:
Due to `CONFIG_BT_MESH_GATT_PROXY` also being set in overlay_pst, where `CONFIG_BT_MESH_PROXY_SOLICITATION` is **not** set, it is necessary to check for both conditions (or at least the latter) for this test. Should this be changed so that  `CONFIG_BT_MESH_GATT_PROXY` is exclusively set in overlay_gatt? Tests would then need to be updated accordingly.